cpp/src/column/column.cpp

```diff
@@ -212,13 +211,17 @@ namespace{
 }
 
 // returns the size in bytes of the specified gdf_dtype
-gdf_size_type gdf_dtype_size(gdf_dtype dtype) {
-  return cudf::type_dispatcher(dtype, get_type_size{});
+gdf_size_type gdf_dtype_size(gdf_dtype dtype)
+{
+  return cudf::size_of(dtype);
 }
+
 // Returns the size in bytes of the data type of the gdf_column
 gdf_error get_column_byte_width(gdf_column * col, 
```

Comment:
Likewise I'd think this should be excised in favor of `cudf::byte_width`

Reply:
Again, can't be done since it's an API function :-(

I know making the API C++'ish is decided, but actually - it seems we haven't considered changes such as exposing some of the `namespace cudf` functionality outwards.